### ISSUE TITLE
[BUG] KeyError: `PWN_RELEASE'

### ISSUE THREAD

**Issue by amooz** [aluminum-ice/pwnagotchi#85]
**Describe the bug**
I downloaded and flashed the freshly released v1.8.2 onto a brand new SD card this morning and it wouldn't boot.  SSH'ing into the zero 2w, I see the following error when trying to start the pwnagotchi service:

```bash
root@pwnagotchi:/var/log # pwnagotchi
Traceback (most recent call last):
  File "/usr/local/bin/pwnagotchi", line 9, in <module>
    import pwnagotchi
  File "/usr/local/lib/python3.7/dist-packages/pwnagotchi/__init__.py", line 8, in <module>
    from pwnagotchi._version import __version__
  File "/usr/local/lib/python3.7/dist-packages/pwnagotchi/_version.py", line 3, in <module>
    __version__ = os.environ["PWN_RELEASE"]
  File "/usr/lib/python3.7/os.py", line 678, in __getitem__
    raise KeyError(key) from None
KeyError: 'PWN_RELEASE'
```

**To Reproduce**
1. flash v1.8.2
2. ssh into the device
3. stop the service `sudo systemctl stop pwnagotchi`
4. start the service `sudo pwnagotchi`

**Expected behavior**
Boots without an error

**Screenshots**

**Environment (please complete the following information):**
 - Pwnagotchi version v1.8.2
 - OS version fresh flash
 - Type of hardware rpi zero 2
 - Any additional hardware used none

**Additional context**
none

**Comment by amooz** [comment]
Adding `PWN_RELEASE=1.8.2` to my env got me further into the boot process: 
```bash
Traceback (most recent call last):
  File "/usr/local/bin/pwnagotchi", line 150, in <module>
    pwnagotchi.set_name(config['main']['name'])
  File "/usr/local/lib/python3.7/dist-packages/pwnagotchi/__init__.py", line 44, in set_name
    pwnagotchi.reboot()
NameError: name 'pwnagotchi' is not defined
```

**Comment by amooz** [comment]
Ok, 2 reboots later now it works.  First reboot seems to have created defaults.toml then exploded, the second boot then started working.  I feel like perhaps I missed a step after the initial flashing.

**Comment by aluminum-ice** [comment]
Fixed in release [v1.8.3](https://github.com/aluminum-ice/pwnagotchi/tree/1.8.3)
